lib/oli/utils/db_seeder.ex

```diff
@@ -729,12 +729,6 @@ defmodule Oli.Seeder do
       %Publication{published: nil} = p ->
         Oli.Publishing.update_publication(p, %{published: DateTime.utc_now()})
     end
-
```

Comment:
This is unnecessary now, and I believe was also unnecessary before (view was refreshed by the trigger regardless)